django_mesh/templates/django_mesh/base.html

```diff
@@ -1 +1,16 @@
 {% extends "base.html" %}
+{% block comment %}
```

Comment:
Could you override this file in:
django_mesh_test_project/templates/django_mesh/base.html

We want to show an example of using Disqus but not require it in the importable module.